tsconfig.json

```diff
@@ -7,9 +7,5 @@
         "esModuleInterop": true,
         "noImplicitAny": true,
         "sourceMap": true
-    },
-    "include": [
-        "./src/index.ts",
-        "./node_modules/pixi.js/pixi.js.d.ts"
-    ]
+    }
```

Comment:
The previous comment was only about `"./node_modules/pixi.js/pixi.js.d.ts"`, not `"./src/index.ts"`.
Make sure to read it carefully once more to recall why.